internal/pkg/template/templates/environment/cf.yml

```
@@ -56,8 +56,93 @@ Conditions:
   ManagedAliases: !And
     - !Condition DelegateDNS
     - !Not [!Equals [ !Ref Aliases, "" ]]
+{{- if and .PublicHTTPConfig.ELBAccessLogs (not .PublicHTTPConfig.ELBAccessLogs.BucketName) }} #.PublicHTTPConfig.ELBAccessLogs.CreateBucket
```

What do you think of adding a method instead:
```suggestion
{{- if and .PublicHTTPConfig.ELBAccessLogs.ShouldCreateBucket }} 
```

and in `template/env.go`
```go
func (elb *ELBAccessLogs) ShouldCreateBucket() bool {
   if elb == nil { return false }
   if len(elb.BucketName) > 0 { return false }
   return true
}
```